sage/group_prover.sage

```diff
@@ -100,7 +100,7 @@ class fastfrac:
     """Multiply something else with a fraction."""
     return self.__mul__(other)
 
-  def __div__(self,other):
+  def __truediv__(self,other):
```

Comment:
This breaks sage 8.9. Can you leave the originas `__div__` definition (which shouldn't affect python 3) and add `__truediv__`? I tested that with this change both versions give the same output.

Reply:
I can do this but I don't understand that this helps for sage 8.9 because it's based on python 2 and all the other changes should break it. So you're saying this works for you?

 https://wiki.sagemath.org/Python3-Switch

Reply:
the changes about print, xrange and map are all harmless : they work with good old python2-based sage

Reply:
Ok, I did the change.